app/modules/listings/components/listing_element_show.svelte

```diff
@@ -68,8 +98,43 @@
 
   {#if isCreatorMainUser}
     <div class="buttons-wrapper">
+      {#if isNonEmptyArray(recipientListings)}
+        <Dropdown
+          buttonTitle={i18n('Move this element to another list')}
+          dropdownWidthBaseInEm={25}
+          clickOnContentShouldCloseDropdown={true}
+        >
+          <div
+            slot="button-inner"
+            class="element-button"
+          >
+            <span>
+              {@html icon('arrow-right')}
+              {i18n('Move element to another list')}
+            </span>
+            <span>
+              {@html icon('caret-down')}
+            </span>
+          </div>
+          <div slot="dropdown-content">
+            <div class="menu-section">
+              <div role="menu">
+                {#each recipientListings as recipientListing}
+                  <button
+                    role="menuitem"
+                    on:click={() => updateElementListing(recipientListing._id)}
+                    tabindex="-1"
+                  >
+                    {recipientListing.name}
+                  </button>
+                {/each}
+              </div>
```

Comment:
[non-blocking]
This kind of dropdown menu should ideally be keyboard accessible
```suggestion
              <ul role="menu" use:autofocus>
                {#each recipientListings as recipientListing}
                  <li>
                    <button
                      role="menuitem"
                      on:click={() => updateElementListing(recipientListing._id)}
                    >
                      {recipientListing.name}
                    </button>
                  </li>
                {/each}
              </ul>
```
With
```js
  import { autofocus } from '#app/lib/components/actions/autofocus'
```

Reply:
Definitely, well spotted: done in 530463774